demo/index.html

```diff
@@ -30,33 +34,43 @@
       icons="location place starta stopb bus car train walk">
   </iron-iconset>
 
-  <!-- Now create a bunch of icons using our iconset -->
-  <iron-icon icon="my-icons:location"></iron-icon>
-  <iron-icon icon="my-icons:place"></iron-icon>
-  <iron-icon icon="my-icons:starta"></iron-icon>
-  <iron-icon icon="my-icons:stopb"></iron-icon>
-  <iron-icon icon="my-icons:bus"></iron-icon>
-  <iron-icon icon="my-icons:car"></iron-icon>
-  <iron-icon icon="my-icons:train"></iron-icon>
-  <iron-icon icon="my-icons:walk"></iron-icon>
-  <br>
-  <!-- icons may also be specified by index -->
-  <style>
-    .embiggen iron-icon {
-      width: 48px;
-      height: 48px;
-    }
-  </style>
-
-  <div class="embiggen">
-    <iron-icon icon="my-icons-big:0"></iron-icon>
-    <iron-icon icon="my-icons-big:1"></iron-icon>
-    <iron-icon icon="my-icons-big:2"></iron-icon>
-    <iron-icon icon="my-icons-big:3"></iron-icon>
-    <iron-icon icon="my-icons-big:4"></iron-icon>
-    <iron-icon icon="my-icons-big:5"></iron-icon>
-    <iron-icon icon="my-icons-big:6"></iron-icon>
-    <iron-icon icon="my-icons-big:7"></iron-icon>
+  <div class="vertical-section vertical-section-container centered">
+    <!-- Now create a bunch of icons using our iconset -->
+    <h4>Small icon set</h4>
+    <div>
+      <iron-icon icon="my-icons:location"></iron-icon>
+      <iron-icon icon="my-icons:place"></iron-icon>
+      <iron-icon icon="my-icons:starta"></iron-icon>
+      <iron-icon icon="my-icons:stopb"></iron-icon>
+      <iron-icon icon="my-icons:bus"></iron-icon>
+      <iron-icon icon="my-icons:car"></iron-icon>
+      <iron-icon icon="my-icons:train"></iron-icon>
+      <iron-icon icon="my-icons:walk"></iron-icon>
+    </div>
+    <br>
+    <h4>Accessible icon set with labels</h4>
+    <div class="embiggen">
+      <iron-icon icon="my-icons-big:0"></iron-icon>
+      <iron-icon icon="my-icons-big:1"></iron-icon>
+      <iron-icon icon="my-icons-big:2"></iron-icon>
+      <iron-icon icon="my-icons-big:3"></iron-icon>
+      <iron-icon icon="my-icons-big:4"></iron-icon>
+      <iron-icon icon="my-icons-big:5"></iron-icon>
+      <iron-icon icon="my-icons-big:6"></iron-icon>
+      <iron-icon icon="my-icons-big:7"></iron-icon>
+    </div>
+    <br>
+    <h4>Accessible icon set with labels</h4>
+    <div>
+      <iron-icon icon="my-icons:location" tabindex="0" aria-label="icon: location"></iron-icon>
```

Comment:
We took this out of `iron-icon` when we ported it because 1) it resulted in redundant labeling in some cases and 2) it is not trivial to decide what to use as an `alt`, and the 0.5 routine that did this was not very manageable. IMO if a user really wants an icon to have an alt, they should be able to add one easily.